### PR TITLE
fix rendering of Weekly schedules

### PR DESCRIPTION
**Pull request by jacobvenable**
This PR does not close an issue.

## What does this PR do?

Fixes the rendering of weekly schedules in the public section of the app.

example of rendered weekly schedule

_Note: This does not include monthly or annual schedules._

## How does this PR make you feel? [:link:](http://giphy.com/categories/emotions/)

fixed it

### CODE REVIEW

**Comment by rhinodavid** [comment]
Me likey. I'll take a look later today.

File: packages/upswyng-web/src/components/Schedule.tsx [hunk 1]
```diff
@@ -54,78 +54,47 @@ const WeeklyTime = styled.span`
...
+  const dayItemsMap = days.map((day, dayIndex) => {
     return {
       day,
       items: items.filter(
         item =>
           item.recurrenceRule.options.freq === RRule.WEEKLY &&
-          item.recurrenceRule.options.byweekday.includes(
-            (RRule as any)[dayToCode[day as keyof typeof dayToCode]] as number
-          )
+          item.recurrenceRule.options.byweekday.includes(dayIndex)
```
Comment (rhinodavid) [inline packages/upswyng-web/src/components/Schedule.tsx]: logic error around hurr

File: packages/upswyng-web/src/components/Schedule.tsx [hunk 2]
```diff
@@ -138,12 +107,12 @@ const Schedule = ({ schedule }: ScheduleProps) => {
     // group together schedule items that have the same comments
     const commentMap = resourceSchedule.getItems().reduce((result, item) => {
       const comment = item.comment || "_no_comment_";
-      result[comment as keyof typeof dayToCode] = [
-        ...(result[comment as keyof typeof dayToCode] || []),
+      result[comment as keyof typeof days] = [
```
Comment (rhinodavid) [inline packages/upswyng-web/src/components/Schedule.tsx]: i think you can prob get rid of all this type info if you're just gonna use an array for `days`.. `keyof typeof days` reduces to `number`

Comment (rhinodavid) [reply packages/upswyng-web/src/components/Schedule.tsx]: TS may fight you tho

**Comment by rhinodavid** [comment]
Fixed the whole off-by-a-day thing by going back to the RRule code. Still needs a little work though I think.

Screen Shot 2020-01-05 at 4 42 43 PM
Screen Shot 2020-01-05 at 4 42 34 PM

File: packages/upswyng-web/package.json [hunk 3]
```diff
@@ -9,7 +9,7 @@
     "@material-ui/lab": "^4.0.0-alpha.27",
     "@types/algoliasearch": "^3.30.12",
     "@types/enzyme": "^3.9.1",
-    "@types/enzyme-adapter-react-16": "^1.0.5",
+  "@types/enzyme-adapter-react-16": "^1.0.5",
```
[the remark this answers is not in the document]
Comment (jacobvenable) [reply packages/upswyng-web/package.json]: Odd, usually prettier fixes these types of things, but I'm not sure why it changed in the first place.

> also already in the root package.json

Our client CI in Netlify only knows of the packages in this file, correct? If so, then that means it needs to be kept, yes?

[the remark this answers is not in the document]
Comment (rhinodavid) [reply packages/upswyng-web/package.json]: Um i _think_ it runs 'yarn' at the root so it should do everything..

[the remark this answers is not in the document]
Comment (rhinodavid) [reply packages/upswyng-web/package.json]: Also I think the prettier setup may be not doing these json files at the roots

File: packages/upswyng-web/src/components/Schedule.tsx (hunk 2 above)
Comment (rhinodavid) [reply packages/upswyng-web/src/components/Schedule.tsx]: Outdated

**Comment by jacobvenable** [comment]
> I'm good with this minus the little package.json spacing thing.

> I think next we need to look at the design of this page. My gripe is that we're doing columns on mobile, which already isn't ideal and wastes a lot of space when just the heading is on the left. And then with the schedule we further subdivide into 'schedule' '' '' and it just seems v crowded.

Agreed. Let me see if I can put together something a little prettier and make another PR. I'll also working on handling output of other formats.

**Pull request merged**